### PR TITLE
Setup hello microverse project

### PR DESCRIPTION
**Pull request by studiosnabeel**
Added linters
Created .gitignore file with test.md and node_modules inside it
Created index.html and style.css

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
...
+Project Name: 
+
+Hello Microverse
+
+Description: 
+
+This project is an exercise for Microverse students to get familiar with git, github, 
+
+github flow and linters for HTML and CSS.
+
```
Comment (ha-manel) [inline README.md]: - [ ] As requested by the previous reviewer, kindly update your README file according to [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) (copy the sections and fill them out with the information of your project) this way your project documentation will be professional ✨

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @studiosnabeel 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️

## Required Changes:
- [ ] From now on, please don't open new pull requests for the same project, you should make the changes on the same pull request and submit it until you get approved 👍 
- [ ] Kindly update the title of this pull request as requested by the previous code reviewer. It should have a descriptive title telling you what changes are inside.
- [ ] Kindly update the description of this pull request as requested by the previous code reviewer. It should have a summary describing introduced changes in more detail.

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by studiosnabeel** [comment]
I need help. Can somebody come on zoom and I'll share screen and take me
through this process once ?

**Comment by studiosnabeel** [comment]
https://github.com/studiosnabeel/Hello/pull/1 kindly check if my pull
request is ok to be approved
regards
Nabeel Akbar

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @studiosnabeel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7


## Highlights :heavy_check_mark: 
- Linter checks are passing
- GitHub flow is correctly used
- Professional README file
- PR has a good title and summary

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**